packages/client/src/subscribe/ordering/OrderedMsgChain.ts

```diff
@@ -182,7 +183,8 @@ class OrderedMsgChain extends MsgChainEmitter {
         gapHandler: GapHandler,
         propagationTimeout = DEFAULT_PROPAGATION_TIMEOUT,
         resendTimeout = DEFAULT_RESEND_TIMEOUT,
-        maxGapRequests = MAX_GAP_REQUESTS
+        maxGapRequests = MAX_GAP_REQUESTS,
+        lighterGapFill = true
```

Comment:
All client defaults are in `config.schema.json`, so there is no need to duplicate the info here.

Should also remove other defaults, e.g.  DEFAULT_PROPAGATION_TIMEOUT (it is the `gapFillTimeout` config option) in some PR.